i18n-scripts/fix-po-files.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sed -i -r "s|plural=(n != 1)|plural=(n != 1);|g" ./po-files/es/plugin__kubevirt-plugin.po
```

Comment:
1. please provide an example of incorrect strings
2. the scrip uses different replace patterns depending on the language - can we make it generic and run all the replacements on all files?  
3. the files are hardcoded - can we use the list of supported locales and calculate them?
4. please rename the script to better reflect it's purpose

Reply:
i made a new script. Please review. 

Wrong:
Plural-Forms: nplurals=2; plural=(n != 1)\n
Correct:
Plural-Forms: nplurals=2; plural=(n != 1);\n